components/report-manager-service/worker/worker.go

```diff
@@ -248,6 +249,28 @@ func (t *GenerateReportTask) Run(ctx context.Context, task cereal.Task) (interfa
 			return nil, err
 		}
 	}
+	lifecycles, err := t.ObjStoreClient.GetBucketLifecycle(ctx, job.RequestToProcess.RequestorId)
+	// fmt.Println(lifecycles)
+
+	if lifecycles == nil && err.Error() == "The lifecycle configuration does not exist" {
```

Comment:
Does the GetBuckeLifeCycle return error if the bucket doesn't have any life cycle configuration?